accounts/serializers.py

```
@@ -1,19 +1,24 @@
 from rest_framework import serializers
 from django.contrib.auth import get_user_model
 from django.contrib.gis.geos import Point
+from django.contrib.auth.password_validation import validate_password
```

Ruff (0.11.9)

`django.contrib.auth.password_validation.validate_password` imported but unused

Remove unused import: `django.contrib.auth.password_validation.validate_password`

(F401)